src/GraphType.jl

```diff
@@ -242,10 +253,13 @@ mutable struct Graph
         extra_uuids = union(keys(reqs), keys(fixed), map(fx->keys(fx.requires), values(fixed))...)
         extra_uuids ⊆ keys(versions) || error("unknown UUID found in reqs/fixed") # TODO?
 
-        data = GraphData(versions, deps, compat, uuid_to_name, verbose = verbose)
+        data = GraphData(versions, deps, compat, uuid_to_name, verbose = verbose)::GraphData
```

Comment:
That this would be needed is quite surprising to me. Any clue on the reason?

Reply:
https://github.com/JuliaLang/julia/issues/25918

Reply:
Ah I see thanks. Maybe also add a note in a comment pointing to that issue then?